Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'json', :git => 'https://github.com/segiddins/json.git', :branch => 'seg-1.7
 
 group :development do
   cp_gem 'claide',                'CLAide'
-  cp_gem 'cocoapods-core',        'Core'
+  gem 'cocoapods-core',           :git => 'https://github.com/igor-makarov/Core.git',
```

Comment:
Should this be reverted?

Reply:
Yes but before merging - otherwise tests fail.